scality_manila_utils/utils.py

```diff
@@ -119,6 +119,22 @@ def process_check(process):
                                    "started".format(process))
 
 
+def fsync_path(path):
+    """
+    Fsync a file or directory.
+
+    :param path: path to file or directory to fsync
+    :type path: string (unicode)
+    """
+    fd = None
```

Comment:
That is indeed a nicer construct, but the builtin `open` and `io.open` can however only operate on normal files and not directores.